### PR TITLE
README updated for independent samples module

### PR DESCRIPTION
**Pull request by runjin326**
### Purpose/implementation Section
Update README.md file to include description of the new "each-cohort" and "all-cohorts" input args that we implemented.

#### What was your approach?
Paragraphs describing these two params are included.

#### What GitHub issue does your pull request address?
https://github.com/PediatricOpenTargets/ticket-tracker/issues/89

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.
Please read to see whether the description is clear enough

#### Is there anything that you want to discuss further?
No

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/independent-samples/README.md [hunk 1]
```diff
@@ -6,23 +6,50 @@ Many analyses that involve mutation frequencies or co-occurence require that all
 However, the PBTA+GMKF data set includes many cases where multiple speciments were taken from a single individual.
 This analysis creates lists of samples such that there are no cases where more than one specimen is included from each individual.
 
-As different analyses may require different sets of data, we actually generate a few different sets, stored in the `results` subdirectory:
+As different analyses may require different sets of data, we actually generate a few different sets, stored in the `results` subdirectory. We also run the analyses based on different 'independent_level', either 'each-cohort' or 'all-cohorts'. When running with 'each-cohort', we call independent samples for each cohort+cancer_type - and same samples in different cohorts are called "independent". When running with 'all-cohorts', we call independent samples regardless of cohort or cancer_type - and same samples in different cohorts are considered the same.
```
Comment (logstar) [inline analyses/independent-samples/README.md]: Could you clarify "same samples in different cohorts"? I thought all `Kids_First_Biospecimen_ID`s are unique, and each `Kids_First_Biospecimen_ID` has only one cohort, so there cannot be any sample in different cohorts.

**Review by logstar — CHANGES_REQUESTED** [review]
Thank you for updating the README! I only have one question about the revised analysis description, which is listed below.

File: analyses/independent-samples/README.md (hunk 1 above)
Comment (runjin326) [reply analyses/independent-samples/README.md]: @logstar, thanks Yuanchao! Sorry by "samples" I meant "participants" - I have specified above.

**Review by logstar — APPROVED** [review]
Thank you for the clarifications @runjin326 .

This PR looks good to me.

**Pull request merged**